python/oneflow/mock_torch/__init__.py

```diff
@@ -217,6 +217,33 @@ def __call__(self, *args, **kwargs):
 
 
 class enable:
+    """
+    mock_torch.enable(lazy=None, verbose=None) -> None
+
+    Enable mock function.
+    When called, all `torch` modules are automatically replaced with `oneflow` modules at runtime without any code changes, unless `mock_torch.disable()` is called.
```

Comment:
```suggestion
    When called, all `torch` modules are automatically replaced with `oneflow` modules at runtime without any code changes, until`mock_torch.disable()` is called.
```

这里是不是用 `until` 更准确。因为我理解是可以这样交替进行的：

```
flow.mock_torc.enable()
...
flow.mock_torc.disable()
...
flow.mock_torc.enable()
...
```